testing/adios2/engine/bp/TestBPWriteProfilingJSON.cpp

```diff
@@ -17,6 +17,7 @@
 #include <stdexcept>
 
 #include <adios2.h>
+#include "adios2/ADIOSTypes.h"
```

Comment:
No need to include this directly since it's already included by `adios2.h`

Reply:
Done.